templates/base.html

```diff
@@ -32,18 +32,8 @@
                   <a class="nav-link" href="/train">Train</a>
                 </li>
                 <li class="nav-item">
-                  <a class="nav-link disabled" href="/">Log out</a>
+                  <a class="nav-link" href="{{ url_for('logout') }}">Log out</a>
```

Comment:
Instead of having the logout button disabled in the nav, we now redirect to the new `logout` route to logout the user.